bazel/envoy_build_system.bzl

```diff
@@ -0,0 +1,39 @@
+# References to Envoy external dependencies should be wrapped with this function.
+def envoy_external_dep_path(dep):
+    return "//external:%s" % dep
```

Comment:
OK, I'll keep this in mind going forward, good to know what's happening in the Bazel world, but will retain due to the multiple options we're supporting for these deps.